app/models/contact.rb

```diff
@@ -0,0 +1,5 @@
+class Contact < ApplicationRecord
+  validates :first_name, :last_name, presence: true
+  
```

Comment:
Validations for the rest of this domain obj attributes (company, email, etc?)

Reply:
We decided to leave the validations this way for now, as only the first and last names are required fields. We talked about potentially revisiting this at a later date if we felt it was needed.

Reply:
We went back and forth on this, only the first name and last name are required, the rest are not. We wanted to get the table up for all to look at and reference so we chose to only do those two for now. Great question and call out. 

We do have on the agenda to add the rest of rest of the attributes, using the allow_blank: true (this allows for "can be omitted if not provided", and adding format clauses such as formatting for a phone number and email.

Reply:
Understood and I agree with the decision here- thanks both.